firebase/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+''' Builds and exports Firebase dependencies.
+To reference these dependencies, add '//firebase:crashlytics_lib' and '//firebase:crashlytics_deps'
+to your build rule's dependency list.
+'''
+load("@rules_jvm_external//:defs.bzl", "artifact")
```

Comment:
Single line space between file documentation and first line of code

Reply:
Done!